raiden-dapp/src/utils/logstore.ts

```diff
@@ -98,14 +99,19 @@ export async function setupLogStore(additionalLoggers: string[] = ['matrix']): P
 
   for (const log of [logging, ...additionalLoggers.map(logging.getLogger)]) {
     const origFactory = log.methodFactory;
-    log.methodFactory = function raidenMethodFactory(methodName, level, loggerName) {
+
+    function raidenMethodFactory(
+      methodName: string,
+      level: LogLevelNumbers,
+      loggerName: string | symbol,
+    ): LoggingMethod {
       const rawMethod = origFactory(methodName, level, loggerName);
       const name = loggerName.toString();
       setDbForLogger(name);
 
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       return (...message: any[]): void => {
-        rawMethod(...message);
+        rawMethod.apply({}, [...message]);
```

Comment:
Nitpick: 
```suggestion
        rawMethod.apply({}, message);
```

Reply:
This is intended. Else I get an error that instead of one or two arguments I provide one or more.

Reply:
This would be just an optimization, `[...message]` is just creating a shallow copy of the arguments array. Shallow copies are cheap, the nitpick was just to avoid it. The error can be fixed if you type the params as `(...message: [any, ...any[]])` or something like that, but don't mind.

Reply:
I have chosen to go for the `call` variant as some code above changed and also uses the `call` variant already in the CLI.
Done. ✔️